.yamllint

```diff
@@ -0,0 +1,8 @@
+---
+
+extends: default
+
+rules:
+  indentation:
+    level: error
+    indent-sequences: false
```

Comment:
I'd probably move adding a line ending at EOF here to the commit that introduces this file and only keep the formatting changes to the pre-existing files in a separate commit.

Reply:
Sorry, but I don't understand the idea of this. Also, what do you mean about "pre-existing files in a separate commit?"

Reply:
Your last commit changed this config adding one byte at the end of the file. Just move that byte to the first commit.

Reply:
Thanks. It was fixed.

Reply:
Marking this as resolved.